client/src/lib/TinyMCEActionRegistrar.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Acts as top level action registrar component for tinymce
+ */
+class TinyMCEActionRegistrar {
```

Comment:
Yeah, I guess Register would be the appropriate name for it

Sticking to a standard for this would be nice 👍